features/outlines.yml

```diff
@@ -0,0 +1,22 @@
+name: Outlines
+description: The `outline` CSS property sets the color, style, and width of a line around an element, outside of the border. The `outline-offset` CSS property sets the distance from the element's edge to the outline.
+spec: https://drafts.csswg.org/css-ui-4/#outline
```

Comment:
It'd be nice to leave a `TODO` comment here saying we _could_ reference https://caniuse.com/outline but the differences in support need reconciliation.